docs/using.rst

```diff
@@ -105,47 +113,57 @@ To use ADB with Host-Based tools
 
     config.servicemanager.services = 'openshift'
 
-
    **Note:**
-
-  * Docker is a default service for ADB boxes and does not require any configuration to ensure it is started. Additionally, Red Hat Enterprise Linux Container Development Kit boxes, which are based on the Atomic Developer Bundle, also, automatically start OpenShift.
-  * You can enable multiple services as a comma separated list. Eg: `docker, openshift`.
-
+* Docker is a default service for ADB boxes and does not require any configuration to ensure it is started.
+  Additionally, Red Hat Enterprise Linux Container Development Kit boxes, which
+  are based on the Atomic Developer Bundle, automatically start OpenShift as well.
+* You can enable multiple services as a comma separated list. For instance: `docker, openshift`.
 
 3. Enable any specific options for the services you have selected as:
 
- * OpenShift: Specific versions can be specified using the following variables:
+   For instance, in OpenShift, specific versions can be specified using the following variables:
 
    1. `config.servicemanager.openshift_docker_registry = "docker.io"` - Specifies the registry from where the service should be pulled.
```

Comment:
config.servicemanager.openshift_docker_registry = "docker.io" and similar lines are not rendered in coded for as:
`config.servicemanager.openshift_docker_registry = "docker.io"`. The are being rendered as simple text.